src/launch_common.py

```diff
@@ -84,8 +84,8 @@ def add_argument_release_override(parser):
 def add_argument_payment_offset(parser):
     parser.add_argument("-O", "--payment_offset",
                         help="Number of blocks to wait after a cycle starts before starting payments. "
-                             "This can be useful because cycle beginnings may be bussy.",
-                        default=0, type=int)
+                             "This can be useful because cycle beginnings may be busy.",
+                        default=30, type=int)
```

Comment:
see comment from last time. The user will not expect an offset by default and we have to use range [-30, XXX] and I would like to use uint from 0 to cycle end for safety